test/TMxpEntityTagHandlerTest.cpp

```diff
@@ -0,0 +1,141 @@
+
+#include <QTest>
+#include "TMxpEntityTagHandler.h"
+#include "TMxpStubClient.h"
+#include <TMxpTagParser.h>
+#include <TMxpTagProcessor.h>
+#include <TMxpProcessor.h>
+
+class TMxpEntityTagHandlerTest : public QObject {
+Q_OBJECT
+
+private:
+
+private slots:
+    QSharedPointer<MxpNode> parseNode(const QString& tagText) const
+    {
+        auto nodes = TMxpTagParser::parseToMxpNodeList(tagText);
+        return nodes.size() > 0 ? nodes.first() : nullptr;
```

Comment:
Is there a `(bool) MxpNode::isEmpty()` method defined which is cleaner than the `!(MxpNode::size() > 0)` construct?